tasties_app/static/css/recipes.css

```diff
@@ -51,3 +51,18 @@
     justify-content: space-between;
     padding: 10px 0 0 0;
 }
+#category_btn:hover{
```

Comment:
I don't know why but all the CSS below doesn't work when I pull this PR.
Is it working for you?